docpages/07_deprecated_list.md

```diff
@@ -1,9 +1,7 @@
 \page deprecated Deprecated List
 
-### Deprecation policy
+# Deprecation Policy
```

Comment:
This should be H2 as the page header is already H1

Reply:
Yup, that's correct. Either an oversight or unintentional extra click on backspace on my end it seems. I'll rectify that in the next iteration.